### ISSUE TITLE
Add credits, ratings to users/businesses. More they answer and comment more they will get credit points. More the number of votes more will be the credit accumulation.  

### ISSUE THREAD

**Issue by vedanuj** [vedanuj/blinking#4]

**Comment by jahid90** [comment]
Could you clarify the difference between credits and ratings. How do ratings get affected? `Answer and comment` on what ? Posts?

**Comment by vedanuj** [comment]
Rating : This is how we judge the authenticity or the value of an answer to other users. Its like an upvote. Or rather the difference (upvote - downvote). We have to come up with a better equation to calculate that. This feature is open for debate. Another thing we can do is to rate the businesses according to the user affinity towards the business. The more there are answers by users that recommends a business the higher their rating will be.... 

Credits : Credits are for individual users. They get or gain credits when they answer question, give suggestions etc. They can use these credits to ask questions later. Again this point needs to be thought upon. Instead of making the use of credits to ask a question we can offer some other benefits like coupons  etc from businesses.

Hope this clarifies your doubt a bit.

**Comment by jahid90** [comment]
Yes, it does. So basically ratings are for businesses and credits are for individual users. Thanks for the clarification. Cheers.
